cypress/support/commands.ts

```diff
@@ -14,30 +30,36 @@ function waitForRssItems(url, title, retries = 0) {
 
       if (items.length) {
         const episode = Array.from(items)[0];
-        const episodeTitle = episode
-          .getElementsByTagName("title")[0]
-          .textContent.trim();
+        const episodeTitle = episode.getElementsByTagName("title")[0].textContent.trim();
 
         const enclosure = episode.getElementsByTagName("enclosure")[0];
         const enclosureUrl = enclosure.getAttribute("url");
         const enclosureLength = enclosure.getAttribute("length");
 
+        const image = episode.getElementsByTagName("itunes:image")[0];
+        const imageUrl = image.getAttribute("href");
+
         if (title === episodeTitle) {
           expect(title).to.equal(episodeTitle);
           expect(enclosureLength).to.equal("40557");
         }
 
-        cy.request({
-          url: enclosureUrl,
-          encoding: "binary",
-        }).then((response) => {
+        cy.request({ url: enclosureUrl, encoding: "binary" }).then((response) => {
           cy.writeFile("test.mp3", response.body, "binary");
           cy.readFile("test.mp3", "base64").then((mp3) => {
```

Comment:
Why base64 here?  I ended up reading the file as binary for the image.

Reply:
I think at one point I was going to try to test the actual contents of the file, not just the size